### ISSUE TITLE
item/fileName

### ISSUE THREAD

**Issue by stenskjaer** [scta/edf-schema#15]
Is the `item/fileName` field superfluous in the new EDF system??

**Comment by jeffreycwitt** [comment]
Yes, it should be replaced (or made superfluous) by the consistent use of the `@id` attribute on the `<item>` element.
